PWGCF/FemtoWorld/Tasks/femtoWorldEfficiencyTaskDe.cxx

```diff
@@ -0,0 +1,522 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file femtoWorldEfficiencyTask.cxx
+/// \author Lukasz Graczykowski, WUT Warsaw, lgraczyk@cern.ch
+/// \author Malgorzata Janik, WUT Warsaw, majanik@cern.ch
+/// \author Alicja Plachta, WUT Warsaw, alicja.plachta@cern.ch
+/// \author Barbara Chytla, WUT Warsaw, barbara.chytla@cern.ch
+
+// O2 includes
+#include "Common/DataModel/PIDResponse.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/Core/trackUtilities.h"
+#include "Framework/ASoAHelpers.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+#include "ReconstructionDataFormats/Track.h"
+#include "PWGLF/DataModel/LFResonanceTables.h"
+#include "PWGCF/FemtoWorld/Core/FemtoWorldCollisionSelection.h"
+#include "Framework/O2DatabasePDGPlugin.h"
+
+#include "TPDGCode.h"
+
+using namespace o2;
+using namespace o2::analysis::femtoWorld;
+using namespace o2::track;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+// using TracksPID = aod::FullTracks; // This is okay.
```

Comment:
Please, don't leave code commented

Reply:
I shall correct all mistakes of this kind, not an issue.